pkg/bundle/deploy.go

```diff
@@ -3857,6 +3857,30 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: NooBaaBucketReachingQuantityQuotaState
```

Comment:
@v-harihar notice that deploy.go is generated, don't update it directly. you need to update `File_deploy_internal_prometheus_rules_yaml`

Reply:
My bad